### ISSUE TITLE
Menu not taking me to the main emulation.

### ISSUE THREAD

**Issue by TerryThor731** [qconrad/intellistar-emulator#52]
I was trying to an emulation of your Intellistar Emulator, And It either malfunctioned or crashed because of the menu not taking me to the emulation.

https://github.com/qconrad/intellistar-emulator/assets/170050316/64ba1050-1e1f-40cf-b0f3-a039b8297f30

**Comment by Dalk21** [comment]
The API key is currently invalid or rate limited it seems, someone needs to push a new one

**Comment by TerryThor731** [comment]
> The API key is currently invalid or rate limited it seems, someone needs to push a new one

Oh, You think the API key is invalid because it needs to push a new one?

**Comment by Dalk21** [comment]
Yeah so either the key is being rate limited (doubt it, it's been broken for a few days) or someone needs to edit the code with a new one. I'll see if I can get in contact with a contributor about it
